environments/demo/versions.yaml

```diff
@@ -1,18 +1,18 @@
 versions:
-  hocs_audit:
-  hocs_case_creator:
+  hocs_audit: 2.15.1
+  hocs_case_creator: 1.9.1
   hocs_case_migrator:
```

Comment:
Even though we're not deploying it could we have the version here? 

Any future actions we create likely will jump bump this as there'll be a mapping between the code and the deployment.